das/options.go

```diff
@@ -0,0 +1,94 @@
+package das
+
+import (
+	"errors"
+	"fmt"
+	"time"
+)
+
+var (
+	ErrInvalidOptionValue = func(optionName string, value string) error {
```

Comment:
Don't think this needs to be exported. It's _not_ an Err you can use to compare errors against. (The main goal of exported errors is to be comparable so that any user from the outside can check for it).

We can though defined a base error: `ErrInvalidOption = "das: invalid option"`. 
Then, we can wrap it with `fmt.Errorf("%w: value '%s' cannot be %s")`
This way, any outside user would be able to check for the base error via `errors.Is(err, das.ErrInvalidOption)` if he needs to.